### PR TITLE
new alias for command

### PR DESCRIPTION
**Pull request by shinobijolteon**
new alias for .transfersave for 3ds

### CODE REVIEW

File: cogs/assistance-cmds/transfersave.3ds.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Cart to digital save transfer 
 help-desc: Cart to digital version save transfer tutorial
-aliases: ctdtransfer, ctdsave
+aliases: ctdtransfer, ctdsave, savetransfer,
```
Comment (lifehackerhansol) [inline cogs/assistance-cmds/transfersave.3ds.md:4]: trailing comma

**Review by lifehackerhansol — CHANGES_REQUESTED** [review]
Not 100% sure if we want yet another alias but meh

File: cogs/assistance-cmds/twl.3ds.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Fix broken TWL
-url: https://wiki.hacks.guide/wiki/3DS:TWLFix](https://wiki.hacks.guide/wiki/3DS:DS_mode_unbrick#Section_IV,_Part_I_-_Regular_TWLFix
```
Comment (irowiki) [inline cogs/assistance-cmds/twl.3ds.md]: the TWLfix command is now pointing to the DSU page so this in specific is not needed

**Comment by lifehackerhansol** [comment]
I'm closing this PR because it's kind of a mess; please actually respond to review comments, and also make unrelated changes in a separate PR.

**Pull request closed without merging**